Apps/WebClient/src/ClientApp/src/components/modal/verifySMS.vue

```diff
@@ -85,6 +88,28 @@ export default class VerifySMSComponent extends Vue {
         });
     }
 
+    private getTimeout() {
```

Comment:
kinda nit: name starts with get, but does not return anything

Reply:
removed